cmd/calendarsync/main.go

```diff
@@ -102,6 +102,8 @@ func main() {
 }
 
 func Run(c *cli.Context) error {
+	log.Info("started calendarsync")
```

Comment:
What about also printing the version here? Just stating that calendarsync has started feels a bit redundant.